### PR TITLE
HOCS-3087 Updating dockerfile and adding health check file

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

File: scripts/HealthCheck.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * Class used for container health check by docker compose
+ */
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        var client = HttpClient.newHttpClient();
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:8080/actuator/health"))
```
Comment (LiamMacP) [inline scripts/HealthCheck.java:17]: Could we use `server.port` to make this more resilient? https://github.com/UKHomeOffice/hocs-case-creator/blob/a32b52384663ae9b44e1a53942d8f2d0d2914c4f/src/main/resources/application.properties#L5

Comment (gardnersam) [reply scripts/HealthCheck.java]: This file now uses a command line argument to set the end point to check so it is fully customisable

File: scripts/HealthCheck.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Class used for container health check by docker compose
+ * Pass address to be health checked as the single command line argument
+ */
+public class HealthCheck {
```
Comment (philbarton) [inline scripts/HealthCheck.java:13]: case-creator doesn't have any http endpoints. It doesn't have any dependancys so doesn't need the health check.

**Review by philbarton — CHANGES_REQUESTED** [review]
Sorry, late spot. case-creator is message driven only.

File: scripts/HealthCheck.java (hunk 2 above)
Comment (gardnersam) [reply scripts/HealthCheck.java]: I've removed the file from this repo now

**Review by philbarton — APPROVED** [review]
lgtm

**Pull request merged**